docker/.env

```diff
@@ -1,5 +1,5 @@
 
-BESU_VERSION=latest
+BESU_VERSION=21.10.0
```

Comment:
If that doesn't work with `latest` can you raise an issue?

Reply:
the `21.10.6` which is the latest release version works fine so i guess the `latest` is not stable